Games/types/Mafia/roles/cards/Eavesdrop.js

```diff
@@ -0,0 +1,58 @@
+const Card = require("../../Card");
+const { PRIORITY_DAY_MEETING_DEFAULT } = require("../../const/Priority");
+const Message = require("../../../../core/Message");
+module.exports = class Eavesdrop extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Eavesdrop On": {
+                states: ["Day"],
+                flags: ["voting"],
+                action: {
+                    labels: ["eavesdrop"],
+                    priority: PRIORITY_DAY_MEETING_DEFAULT,
+                    run: function () {
+                        this.actor.role.data.stalk = this.target;
+                    }
+                }
+            },
+            "Eavesdropping": {
+                states: ["Night"],
+                flags: ["anonymous", "speech"],
```

Comment:
you need a "voting" flag if you want input type boolean

right now there is no option for the invisible man to vote yes/no

Reply:
a more appropriate input type might be a single-option "End Meeting" , which illuminati can also use